Gems/SurfaceData/Code/Source/TerrainSurfaceDataSystemComponent.cpp

```diff
@@ -168,12 +152,13 @@ namespace SurfaceData
                     const float terrainHeight = terrain->GetHeight(inPosition, AzFramework::Terrain::TerrainDataRequests::Sampler::BILINEAR, &isTerrainValidAtPoint);
```

Comment:
Is the filter type something that should be passed in from GetSurfacePoints()?

Reply:
It's something we could definitely consider for the future, but it's not currently a part of the SurfaceData API.  It wouldn't be a small change to add it, because we'd need to also update all the existing surface providers to support multiple filter types.